README.md

```diff
@@ -1,6 +1,6 @@
-# Google Software Product Sprint
+# Yusuf's SPS Portfolio
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains the projects I'll work on throughout SPS.
 
 To get started:
```

Comment:
Delete this repo setup instructions, since it doesn't apply to those who view your repo

Reply:
I have deleted it